hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java

```diff
@@ -1126,7 +1140,7 @@ private synchronized void addResourceObject(Resource resource) {
    * @throws IllegalArgumentException when more than
    * {@link Configuration#MAX_SUBST} replacements are required
    */
-  public String substituteVars(String expr) {
+  private String substituteVars(String expr) {
```

Comment:
mention in the javadocs that oozie has a subclass with a method of this name, so it must be kept private